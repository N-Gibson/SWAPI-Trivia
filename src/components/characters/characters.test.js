import React from 'react';
import { shallow } from 'enzyme';
import Character from './CharacterCard.js';

describe('Character', () => {

  beforeEach(() => {
    let wrapper = shallow(<Character />)
  })

  it('should match snapshot', () => {
    console.log(wrapper)
  })

  // it('should match snapshot', () => {
  //   console.log(wrapper)
  //   expect(wrapper).toMatchSnapshot();
  // })
})