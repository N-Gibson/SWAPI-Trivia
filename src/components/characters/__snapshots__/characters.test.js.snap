// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character should match snapshot 1`] = `undefined`;

exports[`CharacterCard should match snapshot 1`] = `undefined`;
